src/CartItem.js

```diff
@@ -0,0 +1,25 @@
+export default function CartItem(props) {
+  const{item, decreaseItem, removeFromCart, addItem} = props
+  return (
+    <li key={item.id}>
+      <img
+        class="cart--item-icon"
+        src={`assets/icons/${item.id}.svg`}
+        alt={`${item.name}`}
+      />
+      <p>{item.name}</p>
+      <button 
+      onClick={() => {
+        console.log('clicked -', item.name)
+        decreaseItem(item)
+        removeFromCart(item)
```

Comment:
Very interesting to call the function but stop it running straight away with the `if` if the quantity is high, I didn't think of this